configs/run-wrapper_config.yaml

```diff
@@ -5,9 +5,16 @@ novaseq:
 
 previous_runs_file_path: 'novaseq_runlist.txt'
 
-#igv:
-#  root_path: '/seqstore/webfolders/igv/data/'
-#  GMS-BT: 'barncancer_hg38'
-#  GMS-AL: 'barncancer_hg38'
-#  G21-004: 'susanne.fransson'
+igv:
+  GMS-BT: 'barncancer_hg38'
+  GMS-AL: 'barncancer_hg38'
+  G21-004: 'susanne.fransson'
 
+hg38ref:
+  GMS-BT: 'yes'
+  GMS-AL: 'yes'
```

Comment:
If i understood it correctly these are directly passed as flag values to another script. Even so, I think it's better to use boolean values and handle the translation later on.

Reply:
added this as a to do for later